app/components/Article/article.css

```
@@ -130,3 +130,12 @@ article .footer-comtainer > * {
 h1.teal {
   color: var(--colors-teal-500);
 }
+@media only screen and (max-width: 640px) {
+  article .footer-comtainer {
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+  article .footer-comtainer > div:nth-child(-n + 2) {
+    flex: 1 1;
+  }
+}
```

A couple small typos
```suggestion
@media only screen and (max-width: 640px) {
  article .footer-container {
    flex-direction: row;
    flex-wrap: wrap;
  }
  article .footer-container > div:nth-child(-n + 2) {
    flex: 1 1;
  }
}
```

---

@jrhender I noticed the typo as well.. but it's everywhere, not just in my additions.

---

Ah I see. Better to keep as is then and fix in another PR I suppose.